ci/playbooks/edpm_build_images/run.yml

```diff
@@ -16,3 +16,4 @@
           {%-   endfor %}
           {%- endif %}
           -e @scenarios/centos-9/zuul_inventory.yml
+          --tags packages
```

Comment:
We don't have this tag here everywhere in edpm_build_images role, ex not on these plays: https://github.com/openstack-k8s-operators/ci-framework/blob/d002a528481727d5d673d4053bed943819082f3a/ci_framework/roles/edpm_build_images/tasks/run.yml